skills/csharp/experimental/itsmskill/Extensions/Teams/TaskModule/AdaptiveCardValue.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace ITSMSkill.Extensions.Teams.TaskModule
+{
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// DataContract on AdaptieCardValue
+    /// </summary>
+    public class AdaptiveCardValue<T>
+    {
+        [JsonProperty("msteams")]
+        public object Type { get; set; } = JsonConvert.DeserializeObject("{\"type\": \"task/fetch\" }");
```

Comment:
use FetchType from ActivityEx?

Reply:
removing since it will not be referenced and wont be needed.